src/sentry/models/activity.py

```diff
@@ -129,7 +129,10 @@ class Activity(Model):
     class Meta:
         app_label = "sentry"
         db_table = "sentry_activity"
-        indexes = (models.Index(fields=("project", "datetime")),)
+        indexes = (
+            models.Index(fields=("project", "datetime")),
+            models.Index(fields=("group_id", "datetime")),
```

Comment:
Adding this is needed to remove the 200 limit here:
https://github.com/getsentry/sentry/pull/100881/files#diff-7eca9adbd66f558ce61d8730f677b7d741bd7f8bdfa9a36e1077973f2f355be2R51